webrender/src/device/gfx/device.rs

```diff
@@ -570,6 +572,51 @@ impl<B: hal::Backend> Device<B> {
             Some(FastHashMap::default())
         };
 
+
+        let descriptor_set_layouts = [DescriptorGroup::Default, DescriptorGroup::Clip, DescriptorGroup::Primitive]
+            .iter()
+            .map(|g| {
+                let layouts = match g {
```

Comment:
let's do `layout_bindings = match g { .. }` returning the static slice of set layouts, so that we can have a single `create_descriptor_set_layout` invocation afterwards